### ISSUE TITLE
bincrafters-package-tools do not support the same remote formats as Conan Multi Packager

### ISSUE THREAD

**Issue by madebr** [bincrafters/community#506]
### Description of Problem, Request, or Question
When using Conan Multi Packager, I can set the `CONAN_UPLOAD` to 
`https://api.bintray.com/conan/bincrafters/public-conan@True@bincrafters`

url=https://api.bintray.com/conan/bincrafters/public-conan
ssl=True
name=bincrafters

See supported formats:
https://github.com/conan-io/conan-package-tools/blob/7eb97506482487538d0488463a77c90f5465ee65/cpt/remotes.py#L97-L106

bincrafters-package-tools only supports the url

**Comment by SSE4** [comment]
@uilianries do we have a conclusion about this one?

**Comment by uilianries** [comment]
I'm trying to reproduce the error here: https://travis-ci.org/uilianries/conan-catch2/builds/479997026

Let's wait for Bintray: https://bintray.statuspage.io/

**Comment by uilianries** [comment]
Seems like be a bug from CPT:

```
>>  docker run --rm -v "/home/uilian/Development/bincrafters/conan-protoc_installer:/home/conan/project" -e CONAN_LOGIN_USERNAME="uilianries" -e CONAN_UPLOAD="https://api.bintray.com/conan/uilianries/conan@True@uilianries@True@upload_repo" -e CONAN_USERNAME="uilianries" ...
```

You can see `CONAN_UPLOAD="https://api.bintray.com/conan/uilianries/conan@True@uilianries@True@upload_repo"`, However I've exported `CONAN_UPLOAD="https://api.bintray.com/conan/uilianries/conan@True@uilianries"`

**Comment by uilianries** [comment]
@madebr I've released Bincrafters package tools 0.17.7 with a new fix.

**Comment by madebr** [comment]
Great work! Many thanks!
